app/views/layouts/_footer_links.html.erb

```diff
@@ -2,6 +2,35 @@
 
 <ul class="govuk-footer__inline-list">
   <li class="govuk-footer__inline-list-item">
-    <%= link_to 'Help', 'https://www.gov.uk/help', class: 'govuk-footer__link' %>
+    <%# Destination needs validating %>
+
+    <%= link_to 'Cookies', 'https://www.gov.uk/help/cookies', class: 'govuk-footer__link' %>
+  </li>
+
+  <li class="govuk-footer__inline-list-item">
+    <%# Content needs creating %>
+
+    <%= link_to 'Contact', 'about/contact', class: 'govuk-footer__link' %>
+  </li>
+
+  <li class="govuk-footer__inline-list-item">
+    <%# Content needs creating %>
+
+    <%= link_to 'Feedback', 'about/feedback', class: 'govuk-footer__link' %>
+  </li>
+
+  <li class="govuk-footer__inline-list-item">
+    <%= link_to 'Privacy policy', 'about/privacy', class: 'govuk-footer__link' %>
```

Comment:
We can use the route helpers for the links, i.e. `about_privacy_path`

Reply:
👍 Thank you